controllers/configuration_controller.go

```diff
@@ -897,9 +1010,36 @@ func (meta *TFConfigurationMeta) assembleExecutorVolumes() []v1.Volume {
 	inputTFConfigurationVolume := meta.createConfigurationVolume()
 	tfBackendVolume := meta.createTFBackendVolume()
 	executorVolumes := []v1.Volume{workingVolume, inputTFConfigurationVolume, tfBackendVolume}
-	if meta.GitCredentialsSecretReference != nil {
-		gitAuthConfigVolume := meta.createGitAuthConfigVolume()
-		executorVolumes = append(executorVolumes, gitAuthConfigVolume)
+	SecretOrConfigMapReferences := []struct {
```

Comment:
```suggestion
	secretOrConfigMapVolumes := []struct {
```

Reply:
this is actually a reference to the config map or secret. Have renamed the variable to secretOrConfigMapReferences